README.rst

```diff
@@ -43,14 +43,14 @@ Install the following prerequisites before using symbiflow-examples:
 .. code:: bash
    :name: install-req-ubuntu
 
-   apt update && apt install -y wget
+   apt update && apt install -y wget curl
```

Comment:
Can you split this into a separate pull request?

Reply:
Given that I used curl to get the content of the latest-packages links, I think that this change and the change in the packages retrieval should go in the same PR (which is already open here: https://github.com/SymbiFlow/symbiflow-examples/pull/83)

Reply:
In separate PR